app/Console/Commands/RebuildCardsTable.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Http;
+use JsonMachine\Items;
+use App\Models\Card;
+
+class RebuildCardsTable extends Command
+{
+    protected $signature = 'scryfall:rebuild-cards';
+    protected $description = 'Rebuild the cards table daily using Scryfall bulk data';
+    
+    public function handle()
+    {
+        ini_set('memory_limit', '1G');
+        $this->info('Fetching Scryfall bulk data URL...');
+        $bulkDataUrl = 'https://api.scryfall.com/bulk-data/default-cards';
+        $response = Http::get($bulkDataUrl);
+        
+        if ($response->failed()) {
+            $this->error('Failed to fetch bulk data metadata. Status: ' . $response->status());
+            return 1;
+        }
+        
+        $downloadUrl = $response->json('download_uri');
+        $localFilePath = storage_path('app/scryfall_cards.json');
+        
+        $this->info("Downloading bulk data to {$localFilePath}...");
+        $downloadResponse = Http::withOptions(['sink' => $localFilePath])->get($downloadUrl);
+        
+        if ($downloadResponse->failed()) {
+            $this->error('Failed to download the bulk data file. Status: ' . $downloadResponse->status());
+            return 1;
+        }
+        
+        $this->info('Clearing existing table...');
+        try {
+            DB::table('cards')->truncate();
+        } catch (\Exception $e) {
+            $this->error('Failed to truncate table: ' . $e->getMessage());
+            return 1;
+        }
+        $this->info('Importing new data...');
+        $this->importCards($localFilePath);
+        $this->info('Cards table rebuilt successfully!');
+    }
+    private function importCards($filePath)
+    {
+        $this->info('Streaming and processing JSON array...');
+        $cards = Items::fromFile($filePath);
+        $processed = 0;
+        foreach ($cards as $card) {
+            try {
+                Card::updateOrCreate(
+                    ['id' => $card->id],
+                    [
+                        'name' => $card->name ?? null,
+                        'scryfall_uri' => $card->scryfall_uri ?? null,
+                        'highres_image' => $card->highres_image ?? false,
+                        'image_status' => $card->image_status ?? 'unknown',
+                        'image_uris' => isset($card->image_uris) ? json_encode($card->image_uris) : '{}',
+                        'foil' => $card->foil ?? false,
```

Comment:
similar to above I don't think we need finishes or foil etc

Reply:
I thought this was important?

Reply:
we don't need finishes or foils because we only need one version of the art. displaying multiple versions of the art might be more confusing than helpful since this isn't a collection tool